client/state/jetpack-connect/selectors.js

```diff
@@ -5,18 +5,20 @@ const isCalypsoStartedConnection = function( state, siteSlug ) {
 		return false;
 	}
 	const site = siteSlug.replace( /.*?:\/\//g, '' );
-	if ( state && state[ site ] ) {
+	const sessions = state.jetpackConnect.jetpackConnectSessions;
+
+	if ( sessions && sessions[ site ] ) {
```

Comment:
Looking at the reducer, `sessions` will _always_ be an object (initial state empty object), so the first truthiness check here is unnecessary.